extensions/wikia/PortableInfobox/tests/nodes/NodeDataTest.php

```diff
@@ -39,93 +39,249 @@ public function sourceDataProvider() {
 	}
 
 	/**
-	 * @covers       \Wikia\PortableInfobox\Parser\Nodes\Node::getSourceLabel
-	 * @dataProvider sourceLabelDataProvider
+	 * @covers       \Wikia\PortableInfobox\Parser\Nodes\Node::getSourcesMetadata
+	 * @dataProvider sourcesMetadataDataProvider
 	 *
 	 * @param $markup
 	 * @param $params
 	 * @param $expected
 	 */
-	public function testSourceLabel( $markup, $params, $expected ) {
+	public function testSourcesMetadata( $markup, $params, $expected ) {
 		$node = \Wikia\PortableInfobox\Parser\Nodes\NodeFactory::newFromXML( $markup, $params );
 
-		$this->assertEquals( $expected, $node->getSourceLabel() );
+		$this->assertEquals( $expected, $node->getSourcesMetadata() );
 	}
 
-	public function sourceLabelDataProvider() {
+	public function sourcesMetadataDataProvider() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function "sourcesMetadataDataProvider" has 234 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)